types/jquery.pnotify/index.d.ts

```diff
@@ -192,9 +191,10 @@ interface PNotifyOptions {
      */
     icon?: any;
     /**
-     * The animation to use when displaying and hiding the notice. "none", "show", "fade", and "slide" are built in to jQuery. Others require jQuery UI. Use an object with effect_in and effect_out to use different effects.
+     * The animation to use when displaying and hiding the notice. "none", "show", "fade", and "slide" are built in to jQuery.
```

Comment:
The documentation that goes with this property seems to be outdated, could you update that too?

Reply:
documentation updated